posts/inside-rust/2020-11-12-source-based-code-coverage.md

```diff
@@ -60,16 +60,18 @@ Work on the implementation [began back in April][MCP], and [many PRs
 later][PRs], it’s ready for you to try. All you need is a recent nightly and
 a tool to read the coverage reports.
 
-[Take a look at this guide to get started][guide]. If you spot any issues,
-please [report them]. It’s a huge help!
+The easiest way to get stared with source-based coverage is through 
+[`cargo-llvm-cov`](cargo-llvm-cov).
+
+If you spot any issues, please [report them]. It’s a huge help!
 
 Finally, if you try it out and it works well, we’d also like to hear from
 you! Come by the [Zulip stream] for this change or comment on the [feature
 request].
 
 [MCP]: https://github.com/rust-lang/compiler-team/issues/278
 [PRs]: https://github.com/rust-lang/rust/pulls?q=is%3Apr+author%3Arichkadel+is%3Aclosed+closed%3A%3C2020-11-06
-[guide]: https://doc.rust-lang.org/nightly/unstable-book/compiler-flags/source-based-code-coverage.html
+[cargo-llvm-cov]: https://github.com/taiki-e/cargo-llvm-cov
```

Comment:
This could be a valid link, but I searched the original link on Google which yielded [this](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/source-based-code-coverage.html), which then points to [this](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/instrument-coverage.html). Not sure if those are better at all.

Reply:
Oh ok, will do.

Reply:
It seems probably better to keep the post similar to what the author intended at the time, and just update the link.